haskell-ide-engine.cabal

```diff
@@ -1,5 +1,5 @@
 name:                haskell-ide-engine
-version:             1.0.0.0
+version:             1.1
```

Comment:
note sure, shouldnt we still have `1.1.0.0`? doesnt this matter from a versioning scheme?

Reply:
No, it does not matter. It is implicitly 0.0